### PR TITLE
Issue2:First commit on feature/ecs-setup

### PR DESCRIPTION
**Pull request by hamada-fatma**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  egress_port         = 0
+}
+
+# ECS Module
+module "ecs" {
+  source = "./modules/ecs"
+
+  # Network Inputs
+  subnets         = module.network.subnet_ids
+  security_groups = [module.ecs.lb_sg_id]
```
Comment (Ismail-Hamada) [inline main.tf]: it's already a resource within the same module, why use outputs?

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  # IAM Variables
+  ecs_task_execution_role_name = "ecs-app-execution-role"
+  ecs_task_role_name           = "ecs-app-task-role"
+  execution_role_policy_arn    = "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
+
+  # Load Balancer Variables
+  load_balancer_name = "web-lb"
+  listener_port      = 80
+  target_group_name  = "web-traffic-tg"
```
Comment (Ismail-Hamada) [inline main.tf]: what if we need more than a single target group?

**Review by Ismail-Hamada — CHANGES_REQUESTED** [review]

**Review by Ismail-Hamada — APPROVED** [review]

**Comment by hamada-fatma** [comment]
Issue2: Implement Network and ECS modules

**Comment by hamada-fatma** [comment]
Implement Network and ECS modules

**Comment by hamada-fatma** [comment]
Implement Network and ECS modules

**Pull request merged**